config/application.rb

```diff
@@ -22,5 +22,8 @@ class Application < Rails::Application
     config.generators do |g|
       g.test_framework = :rspec
     end
+
+    config.i18n.available_locales = [ :en, :es ]
```

Comment:
why we are using `es`?

Reply:
Bhaia, I configured I18n for this project. To test out if two different languages are working simultaneously, I used two languages.